backend/src/users/users.controller.ts

```diff
@@ -17,13 +18,15 @@ export class UsersController {
 
   @Get()
   @Roles(Role.ADMIN)
-  async findAll(@Query() query: FindAllQueryDto): Promise<IPaginationResult<User>> {
+  @UsePipes(new ValidationPipe({ whitelist: true, transform: true }))
+  async findAll(@Query() query: FindAllQueryDto, @Query('sort') sort: SortUserDto): Promise<IPaginationResult<User>> {
```

Comment:
Use const variable for sort query key.
store it in the sort-user.dto.ts

Reply:
done